dockerfiles/che/entrypoint.sh

```diff
@@ -244,16 +244,25 @@ init() {
   fi
   ### Are we going to use the embedded che.properties or one provided by user?`
   ### CHE_LOCAL_CONF_DIR is internal Che variable that sets where to load
-  export CHE_LOCAL_CONF_DIR="${CHE_DATA_HOST}/conf"
-  if [ -f "${CHE_LOCAL_CONF_DIR}/che.properties" ]; then
-    echo "Found custom che.properties..."
-    if [ "$CHE_USER" != "root" ]; then
-      sudo chown -R ${CHE_USER} ${CHE_LOCAL_CONF_DIR}
+  # check if we have permissions to create /conf folder.
+  if [ -w / ]; then
+    export CHE_LOCAL_CONF_DIR="/conf"
+    if [ -f "/conf/che.properties" ]; then
+      echo "Found custom che.properties..."
+      if [ "$CHE_USER" != "root" ]; then
+        sudo chown -R ${CHE_USER} ${CHE_LOCAL_CONF_DIR}
+      fi
+    else
+      if [ ! -d /conf ]; then
+          mkdir -p /conf
+      fi
+      echo "Using embedded che.properties... Copying template to ${CHE_LOCAL_CONF_DIR}/che.properties"
+      cp -rf "${CHE_HOME}/conf/che.properties" ${CHE_LOCAL_CONF_DIR}/che.properties
     fi
   else
-    if [ ! -d ${CHE_LOCAL_CONF_DIR} ]; then
-        mkdir -p ${CHE_LOCAL_CONF_DIR}
-    fi
+    echo "WARN: parent dir is not writeable, CHE_LOCAL_CONF_DIR will be set to ${CHE_DATA}/conf"
```

Comment:
that was a mistake, `CHE_DATA_HOST` is a path actual for user PC like `/home/roman/.che/data` but we need path actual for container `/data`